velox/vector/BiasVector.h

```diff
@@ -123,15 +119,6 @@ class BiasVector : public SimpleVector<T> {
     return valueAtFast(idx);
   }
 
-  /**
-   * Loads a SIMD vector of data at the virtual byteOffset given
-   * Note this method is implemented on each vector type, but is intentionally
-   * not virtual for performance reasons
-   *
-   * @param byteOffset - the byte offset to laod from
-   */
-  xsimd::batch<T> loadSIMDValueBufferAt(size_t index) const;
```

Comment:
This API is used internally.

Reply:
If it helps, we could add a macro to conditionally include this code. We'll then use that macro internally (the usage is in legacy code that cannot be modified due to legal reasons.(